packages/flutter/test/widgets/stack_test.dart

```diff
@@ -799,4 +799,22 @@ void main() {
       "'alignment', or an explicit 'textDirection', to the Stack.",
     );
   });
+
+  testWidgets('Can update clipBehavior of IndexedStack',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(IndexedStack(textDirection: TextDirection.ltr));
+    final RenderIndexedStack renderObject =
+        tester.allRenderObjects.whereType<RenderIndexedStack>().first;
```

Comment:
Probably a little more idiomatic: 

```suggestion
        tester.renderObject<RenderIndexedStack>(find.type(IndexedStack));
```

Reply:
Done